permissions/plugin-zos-connector.yml

```diff
@@ -1,10 +1,10 @@
 ---
 name: "zos-connector"
-github: "jenkinsci/zos-connector-plugin"
+github: &GH "jenkinsci/zos-connector-plugin"
 issues:
-  - jira: '20638'  # zos-connector-plugin
```

Comment:
unless you closed out all jira tickets, i recommend leaving it second with read only (maybe its report: false, check the readme) so plugin site will show the older issues.